session/io/io.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package moby.io.v1;
+
+option go_package = "io";
+
+service IOForwarder {
+	rpc ForwardIO(stream Message) returns (stream Message);
+}
+
+message Message {
+	oneof Input {
+		// InitDoneMessage is used from server to client to notify server is ready
+		InitDoneMessage InitDone = 1;
+		// FdMessage used from client to server for input (stdout, stderr) and 
+		// from server to client for output (stdin)
+		FdMessage File = 2;
+		// ResizeMessage used from server to client for terminal resize events
+		ResizeMessage Resize = 3;
+		// SignalMessage is used from server to client to send signal events
+		SignalMessage Signal = 4;
+	}
+}
+
+message InitDoneMessage{
+	string error = 1; // typed errors?
+}
+
+message FdMessage{
+	uint32 Fd = 1; // what fd the data was from
+	bool EOF = 2;  // true if eof was reached
+	bytes Data = 3;
```

Comment:
The max length of `Data` should be documented.

Reply:
Fixed.